data/sql/updates/db_world/2018_10_17_01.sql

```diff
@@ -0,0 +1,149 @@
+(35604, 0, 0, 1, 11, 0, 100, 0, 0, 0, 0, 0, 48, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Arelas Brightstar - On Spawn - Run Script"),
+(35604, 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 53, 1, 35604, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Arelas Brightstar - On Script - Start WP"),
+(35604, 0, 2, 0, 40, 0, 100, 0, 4, 0, 0, 0, 4, 15852, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Arelas Brightstar - On WP 4 Reached - Play Sound"),
+(35604, 0, 3, 0, 40, 0, 100, 0, 5, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Arelas Brightstar - On WP 5 Reached - Say Line 0"),
+(35604, 0, 4, 0, 40, 0, 100, 0, 8, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Arelas Brightstar - On WP 8 Reached - Say Line 1"),
+(35604, 0, 5, 0, 40, 0, 100, 0, 11, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Arelas Brightstar - On WP 11 Reached - Despawn"),
+(-(@GUID+4), 0, 0, 1, 11, 0, 100, 0, 0, 0, 0, 0, 48, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Argent Peacekeeper - On Respawn - Set Active"),
+(-(@GUID+4), 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 53, 1, 355872, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Argent Peacekeeper - Respawn - Start WP"),
+(-(@GUID+4), 0, 2, 0, 40, 0, 100, 0, 11, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Argent Peacekeeper - On WP 11 Reached - Despawn"),
+(-(@GUID+5), 0, 0, 1, 11, 0, 100, 0, 0, 0, 0, 0, 48, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Argent Peacekeeper - On Respawn - Set Active"),
+(-(@GUID+5), 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 53, 1, 355873, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Argent Peacekeeper - Respawn - Start WP"),
+(-(@GUID+5), 0, 2, 0, 40, 0, 100, 0, 11, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Argent Peacekeeper - On WP 11 Reached - Despawn");
+
+DELETE FROM `creature_text` WHERE `entry` IN (35589, 35604);
+INSERT INTO `creature_text` (`entry`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(35589, 0, 0, "The Argent Crusade invites you to test your skills in honorable combat against other heroes of Azeroth at the Argent Tournament.", 12, 0, 100, 0, 0, 0, 35896, 0, "Jaeren Sunsworn"),
+(35589, 1, 0, "The champions of the world's great nations have gathered at the Argent Tournament Grounds in northeastern Icecrown to face all challengers.", 12, 0, 100, 0, 0, 0, 35871, 0, "Jaeren Sunsworn"),
+(35604, 0, 0, "The Argent Crusade invites you to test your skills in honorable combat against other heroes of Azeroth at the Argent Tournament.", 12, 0, 100, 0, 0, 0, 35896, 0, "Arelas Brightstar"),
+(35604, 1, 0, "The champions of the world's great nations have gathered at the Argent Tournament Grounds in northeastern Icecrown to face all challengers.", 12, 0, 100, 0, 0, 0, 35871, 0, "Arelas Brightstar");
+
+
+--
+-- END UPDATING QUERIES
+--
+COMMIT;
+END //
+DELIMITER ;
+CALL updateDb();
+DROP PROCEDURE IF EXISTS `updateDb`;
```

Comment:
please read: https://github.com/azerothcore/azerothcore-wotlk/wiki/Dealing-with-SQL-files#how-to-create-an-sql-update-file